src/styles/card.css

```diff
@@ -0,0 +1,88 @@
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background: var(--white-color);
+  border: 1px solid var(--white-color);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  max-width: 200px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+  align-self: center;
+}
+
+.card__title {
+  color: var(--main-color);
+  margin-bottom: 4px;
+}
+
+.card__pcode {
+  font-weight: 400;
+  font-size: 10px;
+  font-style: normal;
+  line-height: 14px;
+  color: var(--secondary-color);
+  margin-bottom: 16px;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: var(--main-color);
+  font-weight: 400;
+  margin-bottom: 24px;
+}
+
+.card__reviews-count {
+  font-weight: 400;
+  font-size: 10px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary-color);
+}
+
+.card__price-number {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.card__btn {
+  width: 166px;
+  height: 40px;
```

Comment:
the checklist, point 8
also, about with — you can try to make it spread all over the available width of its parent